src/components/menu-bar/menu-bar.css

```diff
@@ -205,7 +205,7 @@
 }
 
 .mystuff > a {
-  background-image: url("/images/mystuff.png");
+  background-image: url("./icon--mystuff.png");
```

Comment:
The path `/images/mystuff.png` doesn't exist anymore there is similar file `icon--mystuff.png`. Is this a legacy path that we haven't received errors before that it was missing?

Reply:
Interesting... it looks like `/images/mystuff.png` is actually part of the `scratch-www` build output. It's the same 1464-byte file as `icon--mystuff.png`, though, so this change makes sense to me.